test/services/beta/beta_rate.test.js

```diff
@@ -0,0 +1,55 @@
+import { expect } from 'chai';
+
+import * as setupPolly from '../../helpers/setup_polly';
+import EasyPostClient from '../../../src/beta/easypost';
+import FilteringError from '../../../src/errors/general/filtering_error';
+import Fixture from '../../helpers/fixture';
+import Rate from '../../../src/models/rate';
+import Util from '../../../src/utils/util';
+
+/* eslint-disable func-names */
+describe('Beta Rate Service', function () {
+  setupPolly.startPolly();
+
+  before(function () {
+    this.client = new EasyPostClient(process.env.EASYPOST_TEST_API_KEY);
+  });
+
+  beforeEach(function () {
+    const { server } = this.polly;
+    setupPolly.setupCassette(server);
+  });
+
+  it('retrieves a list of stateless rates', async function () {
+    const statelessRates = await this.client.BetaRate.retrieveStatelessRates(
+      Fixture.basicShipment(),
+    );
+
+    expect(statelessRates).to.be.an('array');
```

Comment:
Nit: Expecting an array to be an array when the next step is to iterate over the same array is redundant and not necessary. If this wasn't an array, the next line would fail anyway because it couldn't iterate over each item meaning we get this assertion for free

Reply:
Yup. I should have removed this from my takeover, we can remove this in the next PR.

Reply:
We won't remember 😉 I don't mind stamping again if you want to do it now

Reply:
sure, go ahead and re-stamp it😏